layouts/partials/article/root.html

```diff
@@ -10,11 +10,13 @@
 {{ $type := cond (not .Data.Pages) "child" "parent" }}
 
 {{/* Unique id for this article */}}
-{{ $articleId := .Params.id | default .File.Path }}
+{{ $file := .File }}
+{{ if $file}}
+{{ $articleId := .Params.id | default $file.Path }}
```

Comment:
Just thinking of this although it does the same thing (test locally if you do change):
```suggestion
{{ with .File}}
{{ $articleId := .Params.id | default .Path }}
```